data/json/monster_factions.json

```diff
@@ -2,6 +2,7 @@
     {
         "type" : "MONSTER_FACTION",
         "name" : "",
+        "base_faction" : "",
```

Comment:
It is supposed to be explicitly the same as the faction name. The faction name will probably change, now that it is no longer hardcoded.